src/domain/collaboration/agreement/agreement.entity.ts

```diff
@@ -26,9 +23,8 @@ export class Agreement extends BaseAlkemioEntity implements IAgreement {
   @JoinColumn()
   tagset: Tagset;
 
-  constructor(name: string) {
+  constructor() {
     super();
-    this.name = name;
-    this.tagset = new Tagset(RestrictedTagsetNames.DEFAULT);
+    this.tagset = new Tagset();
```

Comment:
this is possibly changing behaviour

Reply:
The `name` column of `Tagset` is annotated with a default:

```typescript
  @Column({ default: RestrictedTagsetNames.DEFAULT })
  name: string;
```

It makes this call and parameter redundant.